examples/development/.env

```diff
@@ -1 +1,14 @@
-LISK_VERSION=next
+# Choose network and version:
+ENV_LISK_NETWORK=devnet
+ENV_LISK_VERSION=1.1.0-alpha.0
+
+# Ports to expose
+ENV_LISK_HTTP_PORT=4000
+ENV_LISK_WS_PORT=5000
+
+# Database configuration
+ENV_LISK_DATABASE=lisk
+ENV_LISK_USER=lisk
+# Change this password before initializing the
+# postgresql container for the first time.
+ENV_LISK_PASSWORD=password
```

Comment:
Shouldn't it be `ENV_LISK_DB_PASSWORD` or similar?

Reply:
For clarity you mean?